policy/policy.proto

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package ossf.scorecard.policy;
+
+option go_package = "github.com/ossf/scorecard/policy";
+
+message CheckPolicy {
+    
+    // Mode definition.
+    enum Mode {
+        DISABLED = 0;
+        ENFORCED = 1;
+    }
+
+    Mode mode = 1;
+    sint32 score = 2;
```

Comment:
`sint32` -> `int32`?

Reply:
I think `sint32` serializes to smaller number of bytes, see https://developers.google.com/protocol-buffers/docs/encoding?csw=1#types, for a negative number `the resulting varint is always ten bytes long`